src/main/java/meteordevelopment/meteorclient/mixin/ClientConnectionMixin.java

```diff
@@ -55,4 +59,12 @@ private void onSendPacketHead(Packet<?> packet, CallbackInfo info) {
     private void onSendPacketTail(Packet<?> packet, CallbackInfo info) {
         MeteorClient.EVENT_BUS.post(PacketEvent.Sent.get(packet));
     }
+
+    @Inject(method = "exceptionCaught", at = @At("HEAD"), cancellable = true)
+    private void exceptionCaught(ChannelHandlerContext context, Throwable throwable, CallbackInfo ci) {
+        if (throwable instanceof IOException && Modules.get().get(AntiPacketKick.class).catchExceptions()) {
+            if (Modules.get().get(AntiPacketKick.class).logExceptions.get()) ChatUtils.warning("Caught exception: %s", throwable);
```

Comment:
This should use `Modules.get().get(AntiPacketKick.class).warning(...);` to better highlight it is coming from this module.